packages/web-frontend/src/containers/UserMenu/index.js

```diff
@@ -148,60 +77,58 @@ const UserMenu = ({
 
   const signInText = isCustomLandingPage ? 'Log in' : 'Sign In / Register';
 
-  let menuItems = [];
-
   const logout = {
-    text: 'Logout',
+    actionText: 'Logout',
     action: onAttemptUserLogout,
   };
   const changePassword = {
-    text: 'Change Password',
+    actionText: 'Change Password',
     action: onToggleChangePasswordPanel,
   };
-  // if is a custom landing page, display only the appropriate items
-  if (isCustomLandingPage) {
-    const visitMainSite = {
-      text: 'Visit tupaia.org',
-      type: 'link',
-      url: window.location.origin,
-    };
-    menuItems = isUserLoggedIn
-      ? [
-          visitMainSite,
-          changePassword,
-          {
-            ...logout,
-            action: onAttemptLandingPageLogout,
-          },
-        ]
-      : [visitMainSite];
-  } else {
-    // otherwise display as usual
-    const viewProjects = {
-      text: 'View projects',
-      action: onOpenViewProjects,
-    };
-    const helpCentre = {
-      text: 'Help centre',
-      type: 'link',
-      url: 'https://beyond-essential.slab.com/posts/tupaia-instruction-manuals-05nke1dm',
-    };
-    menuItems = isUserLoggedIn
-      ? [
-          viewProjects,
-          changePassword,
-          {
-            text: 'Request country access',
-            action: onToggleRequestCountryAccessPanel,
-          },
-          helpCentre,
-          {
-            text: 'Logout',
-            action: onAttemptUserLogout,
-          },
-        ]
-      : [viewProjects, helpCentre];
-  }
+  const isDrawer = isMobile();
+
+  const visitMainSite = {
```

Comment:
I think that this could be more readable and easy to maintain by using a compound component pattern (more info [here](https://kentcdodds.com/blog/compound-components-with-react-hooks)) as I think it's hard to keep track of all the conditionals and props done as an object (eg preText).